bin/perseo

```diff
@@ -81,9 +82,11 @@ function loadConfiguration() {
     if (process.env.PERSEO_ENDPOINT_PORT) {
         config.endpoint.port = process.env.PERSEO_ENDPOINT_PORT;
     }
-    if (process.env.PERSEO_MONGO_HOST) {
-        config.mongo.url = 'mongodb://' + process.env.PERSEO_MONGO_HOST + ':27017/cep';
-        config.orionDb.url = 'mongodb://' + process.env.PERSEO_MONGO_HOST + ':27017/orion';
+    if (process.env.PERSEO_MONGO_ENDPOINT) {
```

Comment:
The problem with this approach is that the user could use `?replicaSet=` in the PERSEO_MONGO_ENDPOINT connection string and, at the same time, a replica set in PERSEO_MONGO_RESPLICASET. If at the end we go ahead with two env vars (PERSEO_MONGO_ENDPOINT and PERSEO_MONGO_RESPLICASET vs. the option of having all them at one point in PERSEO_MONGO_ENDPOINT) at least PERSEO_MONGO_ENDPOINT should be checked to detec that `?replicaSet` is not being used.